src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -177,6 +142,46 @@ private void launchNode(SchedulerDriver driver, Offer offer,
     driver.launchTasks(Arrays.asList(offer.getId()), tasks);
   }
     
+  private ExecutorInfo createExecutor(String taskIdName, String nodeName, String executorName) {
+    int confServerPort = conf.getConfigServerPort();
+    List<Resource> resources = getExecutorResources();
```

Comment:
I would pass `resources` into the method to limit the dependencies this method has and to make it easier to unit test.